apps/web/Dockerfile

```
@@ -13,9 +12,18 @@ RUN pnpm install --frozen-lockfile
 
 COPY apps/web ./apps/web
 
+RUN pnpm --filter @cataloggy/web build
+
+FROM node:20-alpine AS runtime
+
+WORKDIR /app
+RUN apk add --no-cache wget && npm install -g serve@14
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)